Dockerfile

```diff
@@ -28,7 +28,8 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 
 # Use ubi-micro as minimal base image to package the manager binary
 # See https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58
-FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
+# use 9.2 over 9.1.0 due to CVEs (glibc-common-2.34-40.el9_1.1 and libcap-2.48-8.el9)
+FROM registry.access.redhat.com/ubi9/ubi-micro:9.2
```

Comment:
I think we always lock to a particular image - either using a shasum, or by using the full version. But 9.2 is a floating reference, at least I think it is. So this should be `registry.access.redhat.com/ubi9/ubi-micro:9.2-15.1696515526`